Babybaras' Jam!

KnockbackNemo opened the pull request:
Babybaras

Laddie#6142, MaddieL#6256

grizzledwarvet23 committed:
commit so far with game elements. capybara on carpet, snakes, background moving.

grizzledwarvet23 committed:
Revert "commit so far with game elements. capybara on carpet, snakes, background moving."

This reverts commit 4cdddf5206f8da1bfa2a079287535b9f9e3dff1e.

grizzledwarvet23 committed:
Revert "Revert "commit so far with game elements. capybara on carpet, snakes, background moving.""

This reverts commit 9ae2ec065aa1faf255a966766b1ccc0ebc94dbaa.

grizzledwarvet23 committed:
Accidentally commited my game to base, but undid changes. Everything is good now

matthew-zhang-306 committed:
fixed instantiated gameobjects leaking into main scene

matthew-zhang-306 committed:
Merge branch 'main' of https://github.com/texas-egads/mega-micro-2022

KnockbackNemo committed:
Created Babybaras' Magic Bug Snatch Game

KnockbackNemo committed:
Audio fix and Debug cleanup

KnockbackNemo commented:
I'm sorry, it appears that I changed the base files and project settings, but I don't know what I did to change them or how to change them back

KnockbackNemo committed:
Removed TimelineSettings asset

matthew-zhang-306 commented on Assets/7-Babybaras/Scripts/Lassoscript.cs:
```diff
@@ -0,0 +1,95 @@
+        public static bool gotLightningbug = false;
+        public static int throwing = 1; // throwing is 1 is throwing, -1 if pulling
+        public static bool lassoing = false;
+        public static AudioSource bugSounds = Managers.AudioManager.CreateAudioSource();
```
On line 14: Would this still work if execution starts from a main menu, before starting up the minigames? This is a static field, which means that at time time this evaluates, Managers.AudioManager might be unset. Are you able to ensure that this works even if that is the case?

KnockbackNemo replied:
Thanks for pointing that out! I don't know how to test or ensure that it will work, so I just removed the static audio sources and made new non-static ones in other classes instead.

matthew-zhang-306 reviewed (changes requested):
.

KnockbackNemo committed:
PR comment change and bug fix

Created new audio sources instead of using the same static one multiple times to (hopefully) address pull request issues.  Also shifted the position of the firebug to fix a bug.
